nebula/ui/components/VPNConnectionInfoContent.qml

```diff
@@ -120,7 +120,9 @@ VPNFlickable {
                         }
 
                         VPNConnectionInfoItem {
-                            title: VPNCurrentServer.localizedExitCityName
+                            title: serverLocations.isMultipHop
+                            ? VPNCurrentServer.localizedExitCityName
+                            : VPNServerCountryModel.getLocalizedCountryName(VPNCurrentServer.exitCountryCode);
```

Comment:
I would like to get rid of this method. See the other PR. I would like to see: `VPNCurrentServer.localizedExitCountryName`

Reply:
Logged a Jira issue https://mozilla-hub.atlassian.net/browse/VPN-3884